src/CMakeLists.txt

```diff
@@ -5,7 +5,18 @@ include_directories(${PROJECT_SOURCE_DIR}/src)
 set(SOURCE_FILES "benchmark.cc" "colorprint.cc" "commandlineflags.cc"
                  "console_reporter.cc" "csv_reporter.cc" "json_reporter.cc"
                  "log.cc" "reporter.cc" "sleep.cc" "string_util.cc"
-                 "sysinfo.cc" "walltime.cc" "complexity.cc")
+                 "sysinfo.cc" "walltime.cc" "complexity.cc" "counter.cc")
+# Add headers to the list of source files. cmake does not require this,
```

Comment:
D'oh! Indeed, this was meant to go as a separate PR. It got accidentally included as a blanket git add.